### ISSUE TITLE
Microsoft ToDo unverified: "This app may be risky"

### ISSUE THREAD

**Issue by gremms1** [dgsasha/remembrance#39]
Hi there,
After a long search I was very happy to have found your app as it meets all the requirements I have (datetime reminders, microsoft todo support, caldav support (coming up in next release)).
However, when trying to add (corporate) Microsoft ToDo syncing I the below message (see image). Anything that can be done about this on your side or is this only an issue with corporate Microsoft-365 accounts?

![image](https://user-images.githubusercontent.com/73878994/236133219-3f2cfff5-19cf-4f39-b9c7-6dded6beafb7.png)

**Comment by dgsasha** [comment]
Yeah, I was already aware of this. I believe I need to make a website to verify the app. I'll have to look into this more.

**Comment by dgsasha** [comment]
Great news friend! This has been fixed for the next update, thanks to github pages.

Edit: I spoke to soon, I didn't test it yet, and then realized I missed a step. Turns out that step requires having a company registered, so I guess this isn't happening.

**Comment by gremms1** [comment]
> 

Happy when I read the first line. Sad when reading the edit. Well, I guess I'll have to find a different way to sync my todos. Thanks for trying!
